src/solver/simulation/sim_alloc_probleme_hebdo.cpp

```diff
@@ -463,7 +463,7 @@ void SIM_AllocationProblemeHebdo(PROBLEME_HEBDO& problem, int NombreDePasDeTemps
         problem.ResultatsHoraires[k].ValeursHorairesDeDefaillancePositive
           = (double*)MemAlloc(NombreDePasDeTemps * sizeof(double));
         problem.ResultatsHoraires[k].ValeursHorairesDENS
-          = (double*)MemAlloc(NombreDePasDeTemps * sizeof(double)); // adq patch
```

Comment:
Would it be possible to go one step further and not use malloc but new/delete or even self manage structures (array, vector, etc.) ?